core/base/src/TSystem.cxx

```diff
@@ -2492,11 +2482,7 @@ static void R__WriteDependencyFile(const TString &build_loc, const TString &depf
 #else
    TString touch = "echo > "; touch += "\"" + depfilename + "\"";
 #endif
-#ifdef ROOTBINDIR
-   TString builddep = ROOTBINDIR;
-#else
-   TString builddep = TString(gRootDir) + "/bin";
-#endif
+   TString builddep = TROOT::GetBinDir();
```

Comment:
This would probably be better:
   TString builddep = "rmkdepend";
   PrependPathName(TROOT::GetBinDir(), builddep);
   builddep += " \"-f";